tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Mocks/IProjectThreadingServiceFactory.ProjectThreadingService.cs

```diff
@@ -67,6 +67,11 @@ public IDisposable SuppressProjectExecutionContext()
             {
                 JoinableTaskFactory.Run(asyncAction);
             }
+
+            public Task SwitchToUIThread()
+            {
+                return Task.CompletedTask;
+            }
```

Comment:
Is this called? Don't consumers of the interface call an extension method for this?

Reply:
I found this class was already implementing a mock. xUnit  cannot override extension methods. This approach was much easier and simpler.

Reply:
But `SwitchToUIThread` is not on `IProjectThreadingService`, it is an extension method. So the caller likely doesn't call this method. If you put a breakpoint in here, I wouldn't expect it to be hit.

Reply:
Yeah. You are right. I deleted it.